src/App.tsx

```diff
@@ -10,33 +12,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+  const [selectedTabId, setSelectedTabId] = useState('tab-4');
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
 
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
+  const handleTabSelecting = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
 
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${selectedTab.title}`}</h1>
 
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTab={selectedTab}
+        selectedTabId={selectedTabId}
+        onTabSelected={handleTabSelecting}
```

Comment:
```suggestion
      <Tabs
        tabs={tabs}
        selectedTabId={correctTabSelected ? selectedTabId : tabs[0].id}
        onTabSelected={handleTabSelecting}
```

Reply:
> 

Still get an error in one of the tests
![Screenshot 2023-02-16 at 09 23 26](https://user-images.githubusercontent.com/22715618/219296171-b069c954-8b9a-4652-9527-be42601cebf2.png)